baseline.xml

```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="3.11.2@d470903722cfcbc1cd04744c5491d3e6d13ec3d9">
+  <file src="lib/Doctrine/DBAL/Driver/Connection.php">
+    <InvalidParamDefault occurrences="1">
+      <code>int</code>
+    </InvalidParamDefault>
+  </file>
+  <file src="lib/Doctrine/DBAL/Driver/DriverException.php">
+    <MissingImmutableAnnotation occurrences="1">
+      <code>public function getMessage();</code>
+    </MissingImmutableAnnotation>
+  </file>
+  <file src="lib/Doctrine/DBAL/Driver/IBMDB2/DB2Connection.php">
+    <UndefinedConstant occurrences="3">
+      <code>DB2_AUTOCOMMIT_OFF</code>
```

Comment:
Please see if this can be avoided by using the stubs:

https://github.com/doctrine/dbal/blob/6c789b30a46906c5e669158809076d2405ea4fa9/tests/phpstan-polyfill.php#L6-L12

Reply:
Oh great, will try.

Reply:
Using that file with `autoloader=` gives me the following error:  `PHP Fatal error:  Class mysqli_result must implement interface Traversable as part of either Iterator or IteratorAggregate in Unknown on line 0`. I will try to do something with the `<stubs>` tag

Reply:
Alternatively, you may try to include the extensions that dont' have proprietary build dependencies to the docker image that runs Psalm so that they don't need to be stubbed (e.g. `pdo-*`, `mysqli`).

Reply:
Ok it's fixed now :)